samples/OriginalUriBaseIds.sarif

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "properties": {
+    "comment": "This sample demonstrates the use of originalUriBaseIds."
+  },
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SarifSamples"
+        }
+      },
+      "originalUriBaseIds": {
+        "PROJECTROOT": {
+          "description": {
+            "text": "The root directory for all project files."
+          },
+          "properties": {
+            "comment": "The SARIF producer has chosen not to specify a URI for PROJECTROOT. See §3.14.14, NOTE 1, for an explanation."
+          }
+        },
+        "SRCROOT": {
+          "uri": "src",

```

Comment:
>src [](start = 18, length = 3)

Is this legal? shouldn't it end in a forward slash? do we have a hole in our validator (or maybe you haven't authored unit tests for these assets that validate?) btw - if this is a bug it's in our spec as well, which i'm looking at

Reply:
You're right. I've been running `sarif validate` on the sample files, but my installed copy doesn't yet have the "trailing slash" fix. I'll switch to using my private bits.

---
In reply to: [414813884](https://github.com/microsoft/sarif-tutorials/pull/14#discussion_r414813884) [](ancestors = 414813884)

Reply:
... aaaaand I just found a bug in the trailing slash analysis. I'll file an SDK bug and push a PR.

---
In reply to: [414824105](https://github.com/microsoft/sarif-tutorials/pull/14#discussion_r414824105) [](ancestors = 414824105,414813884)

Reply:
Filed and fixed the SDK bug, raised a PR on the SDK, added the slash here.

---
In reply to: [414829741](https://github.com/microsoft/sarif-tutorials/pull/14#discussion_r414829741) [](ancestors = 414829741,414824105,414813884)